pkg/hostpath/constants.go

```diff
@@ -1,12 +1,22 @@
 package hostpath
 
-const (
-	deviceID           = "deviceID"
-	maxStorageCapacity = tib
-)
-
 type accessType int
 
 const (
-	mountAccess accessType = iota
+	deviceID                             = "deviceID"
+	kib                       int64      = 1024
+	mib                                  = kib * 1024
+	gib                                  = mib * 1024
+	tib                                  = gib * 1024
```

Comment:
These abbreviations read like `bits` to me instead of `bytes`. And I'm not sure where the `mib` and `tib` came from, since there are not any `i`s in them. How about something like:
```
storageCapacityKB
storageCapacityMB
storageCapacityGB
storageCapacityTB
``` 
It is a bit longer and more verbose, but also more recognizable as you read through the code.

Reply:
these are more carry over from the upstream k8s repo's used as a starting point for this driver

but sure I can rename them as you suggested